COPYING.TXT

```diff
@@ -0,0 +1,347 @@
+
```

Comment:
I think we were talking about more recent license text for GPL2?

Reply:
Replaced